pkg/vault/test/foundry/DynamicFeePoolTest.t.sol

```diff
@@ -107,7 +107,7 @@ contract DynamicFeePoolTest is BaseVaultTest {
 
     function testSwapTooSmallAmountCalculated() public {
         // Near 100% swap fee will result in near 0 amount calculated
-        PoolHooksMock(poolHooksContract).setDynamicSwapFeePercentage(FixedPoint.ONE - 1);
+        PoolHooksMock(poolHooksContract).setDynamicSwapFeePercentage(100e16 - 1);
```

Comment:
I think it's fine to use FP.ONE in the code for percentages. For things like 1-5-10-20% I'd use `e16`